### PR TITLE
Updates to the FlightControl deployment

### PR DESCRIPTION
**Pull request by ajoslin103**
The Flightcontrol has changed a little bit - and Redwood has better support

I got my RW deployed and wanted to update the Docs

### CODE REVIEW

**Review by Tobbe — COMMENTED** [review]
Thanks a lot for your PR @ajoslin103

File: docs/docs/deploy/flightcontrol.md [hunk 1]
```diff
@@ -9,16 +9,17 @@ description: How to deploy a Redwood app to AWS via Flightcontrol
...
+4. If you don't have an account, sign up at [app.flightcontrol.dev/signup](https://app.flightcontrol.dev/signup?ref=redwood)
+5. Create a new project from the onboarding screen or project list
+    1. Connect your Github account and select your repo
+    2. Click "Create Project" and complete any required steps like linking your AWS account.
+    3. Configuration Type should autoselect as `flightcontrol.json`
+    4. NOTE: `flightcontrol.json` is now the source of truth for your Project and it's Environments 
+6. Add your env vars in Flightcontrol for your Environment.
+7. If using dbAuth, add the session secret key env variable in the Flightcontrol dashboard
 
 NOTE: If you are using yarn v1, remove the `installCommand`'s from flightcontrol.json
```
[the remark this answers is not in the document]
Comment (ajoslin103) [reply docs/docs/deploy/flightcontrol.md]: 👍

File: docs/docs/deploy/flightcontrol.md [hunk 2]
```diff
@@ -9,16 +9,17 @@ description: How to deploy a Redwood app to AWS via Flightcontrol
...
 1. In your project, run the command `yarn rw setup deploy flightcontrol --database=YOUR_DB_TYPE` where YOUR_DB_TYPE is `mysql` or `postgresql`
-2. Commit the changes and push to github
-3. If you don't have an account, sign up at [app.flightcontrol.dev/signup](https://app.flightcontrol.dev/signup?ref=redwood)
-4. Create a new project from the onboarding screen or project list
-  1. Connect your Github account and select your repo
-  2. Select "Config Type" as `flightcontrol.json`
-  3. Click "Create Project" and complete any required steps like linking your AWS account.
-5. If using dbAuth, add the session secret key env variable in the Flightcontrol dashboard
-
+2. NOTE: The RedwoodJS Flightcontrol setup defaults to an environment[0] of 'Development'
```
[the remark this answers is not in the document]
Comment (ajoslin103) [reply docs/docs/deploy/flightcontrol.md]: Nope!  Nice catch!

File: docs/docs/deploy/flightcontrol.md [hunk 3]
```diff
@@ -4,21 +4,20 @@ description: How to deploy a Redwood app to AWS via Flightcontrol
 
 # Deploy to AWS with Flightcontrol
 
-[Flightcontrol](https://www.flightcontrol.dev?ref=redwood) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed. It supports servers, static sites, and databases which makes it a perfect fit for hosting scalable Redwood apps.
+[Flightcontrol](https://www.flightcontrol.dev) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed. It supports servers, static sites, and databases which makes it a perfect fit for hosting scalable Redwood apps.
```
Comment (Tobbe) [inline docs/docs/deploy/flightcontrol.md:7]: I'd leave the `ref` in there. IIRC someone from flightcontrol first wrote these docs, so they probably want that to better know where their traffic is coming from 
```suggestion
[Flightcontrol](https://www.flightcontrol.dev?ref=redwoodv) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed. It supports servers, static sites, and databases which makes it a perfect fit for hosting scalable Redwood apps.
```

File: docs/docs/deploy/flightcontrol.md [hunk 4]
```diff
@@ -4,21 +4,20 @@ description: How to deploy a Redwood app to AWS via Flightcontrol
...
-
-NOTE: If you are using yarn v1, remove the `installCommand`'s from flightcontrol.json
+2. NOTE: The RedwoodJS Flightcontrol setup defaults to an environment\[0] of 'Development', add to the array to deploy more environments.
+3. Commit the changes and push to github.
+4. If you don't have an account, sign up at [app.flightcontrol.dev/signup](https://app.flightcontrol.dev/signup?ref=redwood).
+5. Create a new project from the onboarding screen or project list.
+    1. Connect your Github account and select your repo.
+    2. Click "Create Project" and complete any required steps like linking your AWS account.
+    3. Configuration Type should autoselect as `flightcontrol.json`.
+    4. NOTE: `flightcontrol.json` is now the source of truth for your Project and it's Environments.
```
Comment (Tobbe) [inline docs/docs/deploy/flightcontrol.md:19]:
```suggestion
    4. NOTE: `flightcontrol.json` is now the source of truth for your Project and its Environments.
```

**Review by Tobbe — CHANGES_REQUESTED** [review]
Sorry, I found a couple of more things

**Review by Tobbe — APPROVED** [review]

**Comment by Tobbe** [comment]
Thanks @ajoslin103 🙏

**Pull request merged**